eq-author/cypress/integration/authenticated/validation_spec.js

```diff
@@ -738,7 +738,17 @@ describe("Validation", () => {
 
       afterEach(() => {
         closeValidationModal(sidebarButton);
-        removeAnswers();
+        cy.get(testId("btn-delete-answer")).should("have.length", 2);
```

Comment:
Could we put this in the `removeAnswers` function as otherwise isn't there potential for this bug to appear in all of the other `afterEach`'s

Reply:
@pricea1 can you confirm that the new change resolves @SamGodwin2's issue and dismiss his review if true

Reply:
Good for me